### DISCUSSION TITLE
邮件发件服务器连接超时

### DISCUSSION THREAD

**Discussion by Chun-QiuCC** [bs-community/blessing-skin-server#491] [Q&A]
发送邮件失败
以下是报错信息：
`邮件发送失败，详细信息：Connection could not be established with host smtp.exmail.qq.com :stream_socket_client(): Unable to connect to ssl://smtp.exmail.qq.com:465 (Connection timed out)`
已更换多个smtp服务器测试，均为连接超时
加密方式ssl以及tls均尝试过，25端口以及465端口也都尝试过，无一例外全部报错连接超时

**Comment by ColdTearsYY** [comment]
看一下是不是服务商封了，我这有封25，465，587，994的

**Reply by Chun-QiuCC** [reply]
已向服务商核实并未封禁465端口，防火墙/安全组也放行465端口了，依旧报错：
`邮件发送失败，详细信息：Connection could not be established with host smtp.exmail.qq.com :stream_socket_client(): Unable to connect to ssl://smtp.exmail.qq.com:465 (Connection timed out)`
连接的发件服务器是QQ企业邮箱的smtp服务器

**Comment by ColdTearsYY** [comment]
e 建议你还是自己运行一下指令
telnet smtp.exmail.qq.com 465
自己测试看看吧
